src/Nox.Generator/Application/Commands/DeleteRelatedCommand.template.cs

```diff
@@ -0,0 +1,90 @@
+﻿﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Nox.Application.Commands;
+using Nox.Solution;
+using Nox.Types;
+using Nox.Application.Factories;
+using {{codeGeneratorState.PersistenceNameSpace}};
+using {{codeGeneratorState.DomainNameSpace}};
+using {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+using {{entity.Name}}Entity = {{codeGeneratorState.DomainNameSpace}}.{{entity.Name}};
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.Commands;
+
+public partial record DeleteAll{{relationship.Name}}For{{parent.Name}}Command({{parent.Name}}KeyDto ParentKeyDto) : IRequest <bool>;
```

Comment:
It should be used {{GetNavigationPropertyName parent relationship}} instead of {{relationship.Name}}